shaders/palette.frag

```diff
@@ -1,17 +1,36 @@
 #version 330 core
 
 in vec2 tex_coord;
+in vec2 fbo_coord;
+in float blend_mode;
```

Comment:
You can pass integers to shaders with `glVertexAttribIPointer`.

Reply:
The current pushed stuff is heavily WIP and getting hacked at as we figure out the actual tags, so none of this is likely to survive that.  This pr is going to get messy for a while. This stuff should eventually get fixed, but not now :P